Sniffs/PHP/NewKeywordsSniff.php

```diff
@@ -142,38 +177,44 @@ public function register()
      */
     public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
     {
-        $tokens = $phpcsFile->getTokens();
+        $tokens    = $phpcsFile->getTokens();
+        $tokenType = $tokens[$stackPtr]['type'];
+
+        // Translate T_STRING token if necessary.
+        if ($tokens[$stackPtr]['type'] === 'T_STRING') {
+            $content = $tokens[$stackPtr]['content'];
+            if (isset($this->translateContentToToken[$content]) === false) {
+                // Not one of the tokens we're looking for.
+                return;
+            }
+
+            $tokenType = $this->translateContentToToken[$content];
+        }
 
         $nextToken = $phpcsFile->findNext(PHP_CodeSniffer_Tokens::$emptyTokens, ($stackPtr + 1), null, true);
         $prevToken = $phpcsFile->findPrevious(PHP_CodeSniffer_Tokens::$emptyTokens, ($stackPtr - 1), null, true);
 
+
         // Skip attempts to use keywords as functions or class names - the former
-        // will be reported by FrobiddenNamesAsInvokedFunctionsSniff, whilst the
-        // latter doesn't yet have an appropriate sniff.
+        // will be reported by ForbiddenNamesAsInvokedFunctionsSniff, whilst the
+        // will be (partially) reported by the ForbiddenNames sniff.
```

Comment:
The sentence doesn't seem to make sense anymore. It used to read 'whilst the latter doesn't yet have...' but now reads 'whilst the will be (partially) reported...'. I'm guessing the word latter is missing ?